scripts/s3.py

```diff
@@ -41,13 +41,22 @@
 def wrap_vault(
     vault: Union[VaultV1, VaultV2], samples: ApySamples, aliases: dict, icon_url: str, assets_metadata: dict
 ) -> dict:
-    apy = vault.apy(samples)
-
+    apy_error = Apy("error", 0, 0, ApyFees(0, 0), ApyPoints(0, 0, 0))
+    try:
+        apy = vault.apy(samples)
+    except ValueError as error:
+        logger.error(error)
+        apy = apy_error
+    except PriceError as error:
+        logger.error(error)
+        apy = apy_error
     new = False
     if isinstance(vault, VaultV2):
         harvests = [harvest for strategy in vault.strategies for harvest in strategy.harvests]
         if len(harvests) < 4:
             new = True
+            apy.net_apy = 0.0
```

Comment:
Fixes a sorting issue on yearn.finance where vaults that display "NEW" are not always sorted uniformly